package.json

```diff
@@ -1490,14 +1490,14 @@
                 "mac": "alt+right",
                 "win": "ctrl+right",
                 "linux": "ctrl+right",
-                "when": "calva:keybindingsEnabled && editorLangId == clojure && editorTextFocus && paredit:keyMap =~ /original|strict/"
+                "when": "calva:keybindingsEnabled && editorLangId == clojure && editorTextFocus && paredit:keyMap =~ /original|strict/ && !calva:cursorInComment || calva:cursorAfterComment"
```

Comment:
@crispinb Seems I get away without the grouping. I don't quite understand why, but I really can't make this fail when I try to.